units/systemd-udevd.service.in

```diff
@@ -28,3 +28,4 @@ MountFlags=slave
 MemoryDenyWriteExecute=yes
 RestrictRealtime=yes
 RestrictAddressFamilies=AF_UNIX AF_NETLINK AF_INET AF_INET6
+SystemCallArchitectures=native
```

Comment:
Do we really want to restrict `systemd-udevd`? See https://github.com/systemd/systemd/issues/5400, https://github.com/systemd/systemd/issues/4293